mpcs/benches/basefold.rs

```diff
@@ -182,7 +187,7 @@ fn bench_batch_commit_open_verify_goldilocks<Pcs: PolynomialCommitmentScheme<E>>
             let values: Vec<E> = evals
                 .iter()
                 .map(Evaluation::value)
-                .map(|x| *x)
+                .copied()
                 .collect::<Vec<E>>();
```

Comment:
(Not part of your PR: if we specify the type in line 187 (on the `let`), we don't need to also specify it here, too.)